### PR TITLE
T181570 Cannot specify deleted/suppressed pages in pages involved field

### PR DESCRIPTION
**Pull request by davidbarratt**

### CODE REVIEW

File: client/src/components/fields/select-pages.js [hunk 1]
```diff
@@ -149,16 +172,20 @@ export class SelectPages extends React.Component {
...
 				value={this.state.value}
 				multi={true}
 				isLoading={loading}
 				options={this.state.options}
-				onInputChange={this.onInputChange.bind( this )}
-				onChange={this.onChange.bind( this )}
-				filterOptions={this.filterOptions.bind( this )}
+				onInputChange={this.onInputChange}
+				onChange={this.onChange}
+				filterOption={option => option}
```
Comment (dayllanmaza) [inline client/src/components/fields/select-pages.js]: `filterOption={option => option}` I think this is the default behavior if I'm not mistaken.

File: client/src/components/fields/select-users.js [hunk 2]
```diff
@@ -143,6 +143,7 @@ export class SelectUsers extends React.Component {
 				options={this.state.options}
 				onInputChange={this.onInputChange.bind( this )}
 				onChange={this.onChange.bind( this )}
+				filterOption={option => option}
```
Comment (dayllanmaza) [inline client/src/components/fields/select-users.js]: Same as above

File: client/src/components/fields/select-pages.js [hunk 3]
```diff
@@ -83,7 +89,7 @@ export class SelectPages extends React.Component {
 		const title = Title.newFromText( text, this.props.site.info );
 
 		return {
-			label: title.getNamespace().isMain() ? title.getKey().replace( /_/g, ' ' ) : `${title.getKey().replace( /_/g, ' ' )} (${title.getNamespace().getNormalizedText()})`,
+			label: title.getNamespace().isMain() ? title.getKey().replace( /_/g, ' ' ) : `${title.getNamespace().getNormalizedText()}:${title.getKey().replace( /_/g, ' ' )}`,
```
Comment (dayllanmaza) [inline client/src/components/fields/select-pages.js]: Nit-pick: you should probably make this multi-line

File: client/src/components/fields/select-pages.js (hunk 1 above)
Comment (davidbarratt) [reply client/src/components/fields/select-pages.js]: it looks like the filtering is _always_ enabled unless you disable it by a function, but I just realized the function should return a bool, so updating. https://github.com/JedWatson/react-select#filtering-options

File: client/src/components/fields/select-users.js (hunk 2 above)
Comment (davidbarratt) [reply client/src/components/fields/select-users.js]: 👍

File: client/src/components/fields/select-pages.js (hunk 3 above)
Comment (davidbarratt) [reply client/src/components/fields/select-pages.js]: it's gone now. :)

**Pull request merged**